### PR TITLE
deactivate Compact Folders fix code

### LINKED ISSUE #114: [BUG] Incorrect behaviour when trying to retrieve/deploy when single child folders are compressed
**Describe the bug**
When a Business Unit folder has only a single metadata type folder when trying to retrieve another metadata type the extension is not able to detect the Business Unit folder click and will retrieve again the single metadata type that already exists. This is due to the fact that Visual Studio Code has enabled in Settings "Explorer: Compact Folders", making the path retrieve to the only configured folder.  

**To Reproduce**
1. Have a single Business Unit in the retrieve folder
2. Retrieve only **one** metadata type. If Compact Folder is enabled the folder will be shown as compacted
3. Try to retrieve another metadata type by right clicking on the same business unit
4. Command will run and retrieve the existing metadata type.

**Expected behavior**
The selection of metadata types popup should be shown everytime the use retrieves/deploys by selecting the Business Unit folder

**Screenshots**
image

**Additional context**
Extension should setup in the settings explorer.compactFolders = false

### PR DESCRIPTION
**Pull request by anasilva105**
# PR details

## What changes did you make? (Give an overview)

Updated Workspace file to consider different settings.
Updated Dependencies file to trigger the explorer.compactFolders to false.

closes #114

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+**
```
Comment (JoernBerkefeld) [inline .prettierignore:1]: what is this for?

File: src/devtools/main.ts [hunk 2]
```diff
@@ -23,6 +23,9 @@ async function initDevToolsExtension(): Promise<void>{
 
         editorDependencies.activateExtensionDependencies(mainConfig.extensionsDependencies);
 
+        // Deactivates Compact folders for command right execution
+        editorDependencies.deactivateCompactFolders();
```
Comment (JoernBerkefeld) [inline src/devtools/main.ts:27]: can you please explain the effect of this for the UI?

File: .prettierignore (hunk 1 above)
Comment (anasilva105) [reply .prettierignore]: pushed by mistake. Removed from branch.

File: src/devtools/main.ts (hunk 2 above)
Comment (anasilva105) [reply src/devtools/main.ts:27]: when a folder has a single folder inside,  VSCode displays it in a line which breaks the execution of the right command. Basically by setting the compact folders to false we are making all the folders to be displayed in a Tree format.

Comment (JoernBerkefeld) [reply src/devtools/main.ts:27]: ok, but thats a rather usefull feature, isn't it? just disabling that might not be what users would like to see?

Comment (anasilva105) [reply src/devtools/main.ts:27]: Below is the difference of what the users will see. I don't think it relevant to have this feature enabled for the end user. Unfortunately, vscode enables it by default. In my code I'm changing it to false on Workspace level, not globally.


Compact Folders = true

image

Compact Folders  = false
image

**Review by JoernBerkefeld — APPROVED** [review]
ok for now - but let's please keep track of the bug ticket for vscode on your end (WATCH) and revert this PR when it becomes possible to keep the setting enabled

**Pull request merged**